using-the-cluster/deploy-prometheus-grafana/track_scripts/setup-crc

```diff
@@ -163,6 +163,17 @@ oc patch -p '{"spec": {"host": "downloads-openshift-console.'$HOSTNAME'.crc.'$IN
 
 oc patch -p '{"spec": {"host": "default-route-openshift-image-registry.'$HOSTNAME'.crc.'$INSTRUQT_PARTICIPANT_ID'.instruqt.io"}}' route default-route -n openshift-image-registry --type=merge
 
+# Deploy the demo metrics app
+mkdir -p /root/prometheus
+curl -s https://raw.githubusercontent.com/openshift-instruqt/instruqt/master/using-the-cluster/deploy-prometheus-grafana/scripts/deploy-prometheus.yaml -o /root/prometheus/deploy-prometheus.yaml
+
+oc new-project metrics-demo
```

Comment:
Careful because the first time at this point this will likely fail due to the fact CRC is not really ready, better to move this part to the first challenge dir into a script called "setup-crc"

Reply:
For when you return from PTO lol, what do you mean by move it to a script? Like /01-prepare/setup-crc ?